### PR TITLE
New demo links

### PR DESCRIPTION
**Pull request by gsheni**
- fix demo links

### CODE REVIEW

File: featuretools/tests/demo_tests/test_demo_data.py [hunk 1]
```diff
@@ -1,29 +1,42 @@
+try:
```
Comment (rwedge) [inline featuretools/tests/demo_tests/test_demo_data.py]: If this is a python2 / 3 thing can we use the `is_python_2` check so it's easy to locate when we remove python2 support?

File: featuretools/tests/demo_tests/test_demo_data.py [hunk 2]
```diff
@@ -1,29 +1,42 @@
...
+
+
+import pytest
+
 from featuretools.demo import load_flight, load_mock_customer, load_retail
 from featuretools.synthesis import dfs
 
 
-def test_load_retail_diff():
+@pytest.fixture(autouse=True)
```
Comment (rwedge) [inline featuretools/tests/demo_tests/test_demo_data.py]: if this fixture is autouse do we need to

File: featuretools/tests/demo_tests/test_demo_data.py (hunk 1 above)
Comment (gsheni) [reply featuretools/tests/demo_tests/test_demo_data.py]: Fixed
https://github.com/Featuretools/featuretools/pull/542/commits/8aef8890243b08a35cf59e4ab31d33405f2e7ee3

File: featuretools/tests/demo_tests/test_demo_data.py (hunk 2 above)
Comment (gsheni) [reply featuretools/tests/demo_tests/test_demo_data.py]: Fixed
https://github.com/Featuretools/featuretools/pull/542/commits/1cdad1be41a7fa99988c36bacc14c43fb8e856c7

**Pull request merged**